Mathlib/Algebra/Star/SelfAdjoint.lean

```diff
@@ -186,7 +186,7 @@ variable [MulOneClass R] [StarMul R]
 
 variable (R)
 
-theorem _root_.isSelfAdjoint_one : IsSelfAdjoint (1 : R) :=
+@[simp] theorem _root_.isSelfAdjoint_one : IsSelfAdjoint (1 : R) :=
   star_one R
```

Comment:
Any chance to add `isSelfAdjoint_ofNat` while we're at it?

Reply:
It's already there as `isSelfAdjoint_natCast`